cli/README.md

```diff
@@ -184,48 +183,138 @@ mainfluxlabs-cli messages read <channel_id> <thing_auth_token>
 ### Groups
 #### Create new group
 ```bash
-mainfluxlabs-cli groups create '{"name":"<group_name>","org_id":"<org_id>","description":"<description>","metadata":{"key":"value",...}}' <user_auth_token>
+mainfluxlabs-cli groups create '{"name":"<group_name>","description":"<description>","metadata":{"key":"value",...}}' <org_id> <user_token>
 ```
+
 #### Delete group
 ```bash
-mainfluxlabs-cli groups delete <group_id> <user_auth_token>
+mainfluxlabs-cli groups delete <group_id> <user_token>
 ```
-#### Get group with id
+
+#### Get group by id
 ```bash
-mainfluxlabs-cli groups get <group_id> <user_auth_token>
+mainfluxlabs-cli groups get <group_id> <user_token>
 ```
+
 #### List all groups
 ```bash
-mainfluxlabs-cli groups get all <user_auth_token>
+mainfluxlabs-cli groups get all <user_token>
+```
+
+#### Update group
+```bash
+mainfluxlabs-cli groups update '{"id":"<group_id>","name":"newName"}' <user_token>
+```
+
+#### List things by group
+```bash
+mainfluxlabs-cli groups things <group_id> <user_token>
+```
+
+#### View group by thing
+```bash
+mainfluxlabs-cli groups thing <thing_id> <user_token>
+```
+
+#### List channels by group
+```bash
+mainfluxlabs-cli groups channels <group_id> <user_token>
+```
+
+#### View group by channel
+```bash
+mainfluxlabs-cli groups thing <channel_id> <user_token>
+```
+
+### Orgs
+#### Create new org
+```bash
+mainfluxlabs-cli orgs create '{"name":"<org_name>","description":"<description>","metadata":{"key":"value",...}}' <user_token>
+```
+
+#### Get org by id
+```bash
+mainfluxlabs-cli orgs get <org_id> <user_token>
+```
+
+#### List all orgs
+```bash
+mainfluxlabs-cli orgs get all <user_token>
+```
+
+#### Update org
+```bash
+mainfluxlabs-cli orgs update-org '{"id":"<org_id>","name":"new_name"}' <user_token>
+```
+
+#### Delete org
+```bash
+mainfluxlabs-cli orgs delete <org_id> <user_token>
 ```
 
 #### Assign user to an org
 ```bash
-mainfluxlabs-cli orgs assign <user_id> <group_id> <user_auth_token>
+mainfluxlabs-cli orgs assign '{"members": ["member_id":"<member_id>","email":"<email>","role":"<role>"]}' <org_id> <user_token>
 ```
+
 #### Unassign user from org
 ```bash
-mainfluxlabs-cli orgs unassign <user_id> <group_id> <user_auth_token>
+mainfluxlabs-cli orgs unassign '{"member_ids":["<member_id>"]}' <org_id> <user_token>
+```
+
+#### Update member
+```bash
+mainfluxlabs-cli orgs update-member '{"id":"<member_id>","email":"<email>","role":"<new_role>"}' <user_token>
```

Comment:
orgs ID is missing and we should use member_id instead of email to identify the member